26-register_publication/443-creator_begins_registering_a_registration_in_the_wizard.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Creator begins registering a Registration in the Wizard
+  @443
+  Scenario Outline: Creator begins registering a Registration in the Wizard
+    Given Creator begins registering a Registration
+    And they have selected one of the <Methods> for starting the Wizard

```

Comment:
```suggestion
    And they have selected <Method> for starting the Wizard
```

Tror man skal kunne bytte ut <Example> med det som står som eksempel. Så det gir ikke mening å si `And they have selected one of the Link to publication for starting the Wizard`